index.js

```diff
@@ -1,6 +1,6 @@
 module.exports = (name) => {
-  const replaceRegex = /[^a-zA-Z0-9-]/g;
-  const replaced = name.replace(replaceRegex, '');
+  const replaceRegex = /[^a-zA-Z0-9-]*(-{2,}?)*(-$)*(^-)*/g;
+  let replaced = name.replace(replaceRegex, '');
```

Comment:
Could you elaborate why replace `const` to `let`? I didn't see that `replaced` is modified afterwards. Better to use `const`.

```suggestion
  const replaced = name.replace(replaceRegex, '');
```